opencti-platform/opencti-graphql/src/domain/container.js

```diff
@@ -143,9 +149,21 @@ export const containersObjectsOfObject = async (context, user, { id, types, filt
   return buildPagination(0, null, resolvedObjects.map((r) => ({ node: r })), resolvedObjects.length);
 };
 
+export const filterUnwantedEntitiesOut = async ({ context, user, ids }) => {
+  const filteredOutInvestigatedIds = [];
+  ids?.forEach((id) => {
+    const entity = internalLoadById(context, user, id);
+    if (!['Task', 'Note'].includes(entity.entity_type)) {
```

Comment:
internalLoadById returns a Promise that needs to be resolved before using it

Reply:
yes, I forgot to update it on linting issues due to the use of a `for...of` loop